dockerfiles/Dockerfile.lint

```diff
@@ -2,15 +2,15 @@ FROM    golang:1.10.3-alpine
 
 RUN     apk add -U git
 
-ARG     GOMETALINTER_SHA=7f9672e7ea538b8682e83395d50b12f09bb17b91
+ARG     GOMETALINTER_SHA=v2.0.6
 RUN     go get -d github.com/alecthomas/gometalinter && \
         cd /go/src/github.com/alecthomas/gometalinter && \
         git checkout -q "$GOMETALINTER_SHA" && \
         go build -v -o /usr/local/bin/gometalinter . && \ 
         gometalinter --install && \
         rm -rf /go/src/* /go/pkg/*
 
-ARG     NAKEDRET_SHA=3ddb495a6d63bc9041ba843e7d651cf92639d8cb
+ARG     NAKEDRET_SHA=v1.0
```

Comment:
I think this is included by default now, so it may not need to be installed here.

Reply:
Thanks for the hint; I can try removing and see if it still works :smile: